### ISSUE TITLE
Suggestion: Permission based class support

### ISSUE THREAD

**Issue by daemitus** [garbagemule/MobArena#20]
Taken right out of the wiki for the config file, adding in a line for permissions so that it would look like (with proper indentation)

classes:
    Knight:
        items: diamond_sword, grilled_pork:2
        armor: 306,307,308,309
        permission: donator
    Archer:
        items: wood_sword, bow, arrow:128, grilled_pork
        armor: 298,299,300,301
        permission: member
    [...]

Basically, the idea is for perk classes for donors and such. At the moment, Ive got them behind lockette doors, however people from time to time still try to bumrush it as someone else opens it up. Thoughts?
